commons/active/opensmile/open_smile_2_audio_analysis.avsc

```diff
@@ -6,6 +6,8 @@
   "fields": [
     {"name": "time", "type": "double", "doc": "Device timestamp in UTC (s)."},
     { "name": "timeCompleted", "type": "double", "doc": "Timestamp in UTC (s) when subject completes the audio questionnaire." },
-    {"name": "data", "type": ["null", "string"], "doc": "Audio features computed by openSMILE recorded in Base64 format.", "default": null}
+    {"name": "config", "type": ["null", "string"], "doc": "OpenSMILE configuration file.", "default": null},
```

Comment:
Perhaps explicitly say config file contents? Or config file identifier, where the questionnaire app makes it completely clear what the possible identifiers are?